### DISCUSSION TITLE
Run Tracking in Parallel

### DISCUSSION THREAD

**Discussion by canozcivelek** [STVIR/pysot#445] [Ideas]
Hi! I'm performing tracking on my Jetson Xavier with an IP camera. I constructed a Gstreamer pipeline to stream which has around around 22 fps. When I start tracking, the fps drops dramatically, resulting in poor performance. I was wondering what tweaks I can do to tackle this issue. Any ideas?

Device: Jetson Xavier
Code: Python 3.6

Thank you!

**Comment by ZhiyuanChen** [comment]
Have you checked out https://github.com/STVIR/pysot/discussions/387 ?

**Reply by canozcivelek** [reply]
Hi thanks for the heads up. Could you explain how I could change the backbone? In the config file there is a field for the backbone but I don't know what "exactly" to do. Is it enough for me to replace the "alexnetlegacy" string with darknet? Any other changes? Appreciate some help. Thanks!

**Reply by ZhiyuanChen** [reply]
> Hi thanks for the heads up. Could you explain how I could change the backbone? In the config file there is a field for the backbone but I don't know what "exactly" to do. Is it enough for me to replace the "alexnetlegacy" string with darknet? Any other changes? Appreciate some help. Thanks!

Sorry for this late reply.  You should put `darknet.py` to appropriate position (i.e. next to `alexnet.py`) first, then replace string of `alexnetlegacy` with `darknet`
